src/main/java/com/gluonhq/substrate/target/MacOSTargetConfiguration.java

```diff
@@ -92,6 +98,11 @@ String getAdditionalSourceFileLocation() {
 
             String staticLibPath = "-Wl,-force_load," + projectConfiguration.getJavafxStaticLibsPath() + "/";
             linkFlags.addAll(asListOfStaticLibraryLinkFlags(staticLibPath, javafxLibs));
+
+            if (projectConfiguration.hasWeb()) {
+                linkFlags.addAll(asListOfStaticLibraryLinkFlags(staticLibPath, macfxWebkit));
```

Comment:
Add the static webkit libraries to the `javafxLibs` list above. Do this after the `if (usePrismSW)` block:

    if (projectConfiguration.hasWeb()) {
        javafxLibs.add(macfxWebkit);
    }

Reply:
ok